src/cli/Utils.cpp

```diff
@@ -177,7 +159,7 @@ namespace Utils
                 slot,
                 blocking,
                 QObject::tr("Please touch the button on your YubiKey to unlock %1").arg(databaseFilename),
-                outputDescriptor));
```

Comment:
@droidmonkey could we make the same change here and pass `quiet` instead of a file descriptor?

Reply:
Possibly but I'll worry about that over in my yubikey PR, there are conflicting changes here.

Reply:
Possibly but I'll worry about that over in my yubikey PR, there are conflicting changes here.